### ISSUE TITLE
文字检测模型文字框一个都检测不出来，请问这是为什么？

### ISSUE THREAD

**Issue by 1wang11lijian1** [PaddlePaddle/PaddleOCR#9484]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
使用PP-OCRv3模型进行文字检测训练，输入图片shape为（1624，1240，3），文字检测模型测试准确率都为0，文字框一个都检测不出来，请问是不是模型输入尺寸和图片大小没匹配上，或者是为什么文字框一个都检测不出来，但是文字识别模型是有效果的。问题#9402 也是我提的问题并没有解决我的疑问

下面是文字检测的训练代码
训练的代码（加载了PP-OCRv3检测模型的权重）
python tools/train.py -c configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_cml.yml -o Global.pretrain_weights=./pretrain_models/en_PP-OCRv3_det_distill_train/

以下是训练的ch_PP-OCRv3_det_cml.yml文件
Global:
debug: false
use_gpu: true
epoch_num: 500
log_smooth_window: 20
print_batch_step: 10
save_model_dir: ./output/en_PP-OCR_v3_det/
save_epoch_step: 100
eval_batch_step:
0
400
cal_metric_during_train: false
pretrained_model: null
checkpoints: null
save_inference_dir: null
use_visualdl: false
infer_img: null
save_res_path: ./output/det/predicts_ppocrv3_en.txt
distributed: true
Architecture:
name: DistillationModel
algorithm: Distillation
model_type: det
Models:
Student:
pretrained:
model_type: det
algorithm: DB
Transform: null
Backbone:
name: MobileNetV3
scale: 0.5
model_name: large
disable_se: true
Neck:
name: RSEFPN
out_channels: 96
shortcut: True
Head:
name: DBHead
k: 50
Student2:
pretrained:
model_type: det
algorithm: DB
Transform: null
Backbone:
name: MobileNetV3
scale: 0.5
model_name: large
disable_se: true
Neck:
name: RSEFPN
out_channels: 96
shortcut: True
Head:
name: DBHead
k: 50
Teacher:
pretrained:
freeze_params: true
return_all_feats: false
model_type: det
algorithm: DB
Backbone:
name: ResNet_vd
in_channels: 3
layers: 50
Neck:
name: LKPAN
out_channels: 256
Head:
name: DBHead
kernel_list: [7,2,2]
k: 50

Loss:
name: CombinedLoss
loss_config_list:

DistillationDilaDBLoss:
weight: 1.0
model_name_pairs:
["Student", "Teacher"]
["Student2", "Teacher"]
key: maps
balance_loss: true
main_loss_type: DiceLoss
alpha: 5
beta: 10
ohem_ratio: 3
DistillationDMLLoss:
model_name_pairs:
["Student", "Student2"]
maps_name: "thrink_maps"
weight: 1.0
model_name_pairs: ["Student", "Student2"]
key: maps
DistillationDBLoss:
weight: 1.0
model_name_list: ["Student", "Student2"]
balance_loss: true
main_loss_type: DiceLoss
alpha: 5
beta: 10
ohem_ratio: 3
Optimizer:
name: Adam
beta1: 0.9
beta2: 0.999
lr:
name: Cosine
learning_rate: 0.001
warmup_epoch: 2
regularizer:
name: L2
factor: 5.0e-05

PostProcess:
name: DistillationDBPostProcess
model_name: ["Student"]
key: head_out
thresh: 0.3
box_thresh: 0.6
max_candidates: 1000
unclip_ratio: 1.5

Metric:
name: DistillationMetric
base_metric_name: DetMetric
main_indicator: hmean
key: "Student"

Train:
dataset:
name: SimpleDataSet
data_dir: ./train_data/icdar2015/text_localization/icdar_c4_train_imgs/
label_file_list:
- ./train_data/icdar2015/text_localization/train_icdar2015_label.txt
ratio_list: [1.0]
transforms:
- DecodeImage:
img_mode: BGR
channel_first: false
- DetLabelEncode: null
- CopyPaste:
- IaaAugment:
augmenter_args:
- type: Fliplr
args:
p: 0.5
- type: Affine
args:
rotate:
- -10
- 10
- type: Resize
args:
size:
- 0.5
- 3
- EastRandomCropData:
size:
- 960
- 960
max_tries: 50
keep_ratio: true
- MakeBorderMap:
shrink_ratio: 0.4
thresh_min: 0.3
thresh_max: 0.7
- MakeShrinkMap:
shrink_ratio: 0.4
min_text_size: 8
- NormalizeImage:
scale: 1./255.
mean:
- 0.485
- 0.456
- 0.406
std:
- 0.229
- 0.224
- 0.225
order: hwc
- ToCHWImage: null
- KeepKeys:
keep_keys:
- image
- threshold_map
- threshold_mask
- shrink_map
- shrink_mask
loader:
shuffle: true
drop_last: false
batch_size_per_card: 10
num_workers: 2
Eval:
dataset:
name: SimpleDataSet
data_dir: ./train_data/icdar2015/text_localization/ch4_test_images/
label_file_list:
- ./train_data/icdar2015/text_localization/test_icdar2015_label.txt
transforms:
- DecodeImage: # load image
img_mode: BGR
channel_first: False
- DetLabelEncode: # Class handling label
- DetResizeForTest:
- NormalizeImage:
scale: 1./255.
mean: [0.485, 0.456, 0.406]
std: [0.229, 0.224, 0.225]
order: 'hwc'
- ToCHWImage:
- KeepKeys:
keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
loader:
shuffle: False
drop_last: False
batch_size_per_card: 1 # must be 1
num_workers: 2

**Comment by LDOUBLEV** [comment]
python tools/train.py -c configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_cml.yml -o Global.checkpoints=./pretrain_models/en_PP-OCRv3_det_distill_train/
这样试试

**Comment by 1wang11lijian1** [comment]
@LDOUBLEV 好兄弟这样改完真的有效果了，非常感谢！但是我有个问题因为本身导入的模型效果就已经很好了，我这边自己训练的究竟提升了多少的效果，有没有可能丝毫没有提升呢？再就是之前导入预训练的模型一直训练不出结果是不是也有可能是这个原因。
